develop/tutorials/articles/01-introduction-to-liferay-development/06-frontend-development-in-liferay/00-frontend-development-in-liferay-intro.markdown

```diff
@@ -17,69 +16,65 @@ technologies you love the most:
 
 ## Loaders [](id=loaders)
 
-To load modules in @product@, you must know when they are needed, where they are 
-located at build time, if you want to bundle them together or load them 
-independently, and you must assemble them at runtime. Keeping track of these 
-tasks can be a hassle. @product@'s Loaders (YUI/AUI and AMD) handle loading for 
-you, so you don't have to worry about all the details. @product@'s loaders are 
-versatile, letting you use YUI/AUI modules, AMD modules, and npm modules 
-(in AMD format). Just provide a small bit of information about your module and 
-@product@'s loaders take care of the rest.
+Ordinarily, loading modules requires you to know when they are needed, where
+they are located at build time, whether you want to bundle them together or load
+them independently, and to assemble them at runtime. Since all that is a hassle,
+@product@'s Loaders (YUI/AUI and AMD) handle it for you. The loaders can use
+YUI/AUI modules, AMD modules, and npm modules (in AMD format). Just provide
+a little information<!--do we have any more detail about this? I hate to leave
```

Comment:
@nealrobbins Yeah we have more info on the loaders. I kept it vague since this was the intro, but I can add some more information if you think it needs it.

Reply:
That's good. Unless it's really simple, I think it might make most sense just to direct readers to the doc that covers loaders in more detail.

Reply:
Sorry, answered below just after answering this one. That tutorial isn't written for 7.1 yet, but exists in 7.0. Once, it's written, I can link to it in the intro.